packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec

```
@@ -11,14 +11,14 @@
 
 %global foreman_proxy_min_version 1.25
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
-%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/lib/foreman-proxy
```

Looks like f825335074aacaf60b97ff99aa72e8b2bc0c8279 introduced this so will need a cherry pick to 2.0. Prior to it, it was `%{_localstatedir}/lib/foreman-proxy`. I think what was meant, was `%{_sharedstatedir}` (see https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/).
```suggestion
%global foreman_proxy_statedir %{_root_sharedstatedir}/foreman-proxy
```

Note that the definition of `_root_localstatedir` should also be changed a few lines above then.